junit5/src/main/java/com/linecorp/armeria/internal/testing/ServerRuleDelegate.java

```diff
@@ -98,6 +107,11 @@ public Server start() {
      */
     public abstract void configure(ServerBuilder sb) throws Exception;
 
+    /**
+     * Configures the {@link WebClient} with the given {@link WebClientBuilder}.
```

Comment:
How about adding a tip?

```suggestion
     * Configures the {@link WebClient} with the given {@link WebClientBuilder}.
     * You can get the configured {@link WebClient} using {@link #webClient()}.
```